README.md

```diff
@@ -12,7 +12,7 @@ at_app is a command-line tool for app developers to quickly generate a starter a
 ### at_app
 
 [![pub package](https://img.shields.io/pub/v/at_app)](https://pub.dev/packages/at_app)
-[![pub points](https://badges.bar/at_app/pub%20points)](https://pub.dev/packages/at_app/score)
+[![pub points](https://img.shields.io/badge/dynamic/json?url=https://pub.dev/api/packages/at_app/score&label=pub%20score&query=grantedPoints)]([https://pub.dev/packages/at_app](https://pub.dev/packages/at_app/score))
```

Comment:
supposed to be this
```md
[![pub points](https://img.shields.io/badge/dynamic/json?url=https://pub.dev/api/packages/at_app/score&label=pub%20score&query=grantedPoints)](https://pub.dev/packages/at_app/score)
```

then it will show as this and can redirect you to the score [![pub points](https://img.shields.io/badge/dynamic/json?url=https://pub.dev/api/packages/at_app/score&label=pub%20score&query=grantedPoints)](https://pub.dev/packages/at_app/score)